scripts/build/Jenkins/Jenkinsfile

```diff
@@ -552,13 +573,15 @@ def CreateSingleNode(Map pipelineConfig, def platform, def build_job, Map envVar
                 }
                 withEnv(GetEnvStringList(envVars)) {
                     def build_job_name = build_job.key
+                    def params = platform.value.build_types[build_job_name].PARAMETERS
                     try {
                         CreateSetupStage(pipelineConfig, snapshot, repositoryName, projectName, pipelineName, branchName, platform.key, build_job.key, envVars, onlyMountEBSVolume).call()
 
                         if(build_job.value.steps) { //this is a pipe with many steps so create all the build stages
                             pipelineEnvVars = GetBuildEnvVars(platform.value.PIPELINE_ENV ?: EMPTY_JSON, build_job.value.PIPELINE_ENV ?: EMPTY_JSON, pipelineName)
                             build_job.value.steps.each { build_step ->
                                 build_job_name = build_step
+                                params = platform.value.build_types[build_job_name].PARAMETERS
```

Comment:
Looks like params on 584 this is an extra copy of params that was moved from 594 to 576.

Reply:
This was because of line 583 which was changing the build_job_name variable to the current build_step

Reply:
I see, it's defined at 576 so it can be used within the finally block. The initial value isn't really required could just use a type or placeholder, but should be fine as-is.

Reply:
Ah, I wasn't sure if the initial value was actually being set to anything. If that's the case, then we can swap it to make it cleaner.

Reply:
I think it should be fine. The build_config file may have parameters set for non-pipe config, so I think it's worth leaving as is. This will ensure it works for both setups. I understand how it's setup now.